### PR TITLE
Docs(Troubleshooting): 21.03 documentation on how to use new `goroutines` option to troubleshoot OOM issues

### PR DESCRIPTION
**Pull request by aaroncarey**
Update to "Troubleshooting" section on OOM issues

- Fixes DOC-178
- Includes some rewording for style, clarity, and ease of translation throughout the file
- Checked for preexisting spelling issues (`aspell check troubleshooting.md`)

### CODE REVIEW

File: content/deploy/troubleshooting.md [hunk 1]
```diff
@@ -1,22 +1,30 @@
...
+During bulk loading of data, Dgraph can consume more memory than usual due to a
+high volume of writes. This can cause OOM crashes.
 
-The recommended minimum RAM to run on desktops and laptops is 16GB.
+You can take the following steps to help avoid OOM crashes:
 
-On EC2/GCE instances, the recommended minimum is 8GB.
+* **Increase the amount of memory available**: If you run Dgraph with insufficient
+memory, that can result in OOM crashes. The recommended minimum RAM to run Dgraph
+on desktops and laptops is 16GB. On EC2 and GCE instances, the recommended minimum
```
Comment (aaroncarey) [inline content/deploy/troubleshooting.md:19]: Is there a reason that we shouldn't say "The recommended minimum ram for desktops, laptops and servers.", to cover on-premises server scenarios?

File: content/deploy/troubleshooting.md [hunk 2]
```diff
@@ -1,22 +1,30 @@
...
 +++
 
-Here are some problems that you may encounter and some solutions to try.
+This page provides tips on how to troubleshoot issues with running Dgraph.
 
-### Running OOM (out of memory)
+### Running out of memory (OOM)
 
-During bulk loading of data, Dgraph can consume more memory than usual, due to high volume of writes. That's generally when you see the OOM crashes.
+During bulk loading of data, Dgraph can consume more memory than usual due to a
```
Comment (karlmcguire) [inline content/deploy/troubleshooting.md:12]: "During bulk loading or backups, Dgraph can consume more memory than usual due to a..."

File: content/deploy/troubleshooting.md (hunk 1 above)
Comment (karlmcguire) [reply content/deploy/troubleshooting.md:19]: I think on headless servers of any kind 8gb is usually enough, so that covers on-premises and EC2/GCE.

**Review by karlmcguire — COMMENTED** [review]
Everything else looks great!

File: content/deploy/troubleshooting.md (hunk 1 above)
Comment (aaroncarey) [reply content/deploy/troubleshooting.md]: Thanks for the clarification; please see updates

File: content/deploy/troubleshooting.md (hunk 2 above)
Comment (aaroncarey) [reply content/deploy/troubleshooting.md]: Please see updates (added links, rewrote to remove passive voice)

**Review by aaroncarey — COMMENTED** [review]
Thanks for your feedback, updated accordingly.

**Review by karlmcguire — APPROVED** [review]

**Pull request merged**